docs/help/getting-started.md

```diff
@@ -1,27 +1,56 @@
 # Deploying Bump.sh
 
-Get started in a few minutes with Bump!
+Get started in a few minutes with Bump.sh!
 
 ## Create your first documentation
-Step 1: Create your space/documentation
-Step 2: Upload your file
-    - upload file directly
-    - use the CLI
-    - try an example
-Step 3: Enjoy
+
+### Step 1: Create your space/documentation
+
+After creating your account, you'll be asked to name your first API documentation.
+By default, your documentation URL will use the same name: you can customize it in the slug field. You can also use [custom domains](help/custom-domains.md), which we'll explain later.
+
+![](/files/doc-creation.png)
+
+### Step 2: Upload your file
+
+If you already have a specification file, you can immediately upload it.
+You can also use our examples to see Bump.sh in action. Finally, our [CLI](help/bump-cli) can also be directly used to jump in.
+
+![](/files/upload-spec-file.png)
+
+### Step 3: Enjoy
+
+Generating your first documentation will take a few seconds. But let's not stop here: many features are awaiting to customize and make the most of your API documentation. Let's have a look in the next section.
+
 
 ## Manage Access
-- Make it public or private
-- Create an organization
-- Invite teammates
+
+Congratulations on releasing your first API documentation on Bump.sh!
+
+One of the first things you may have in mind is how to share your documentation with the world, your team or keep it for yourself.
+Bump.sh offers several options to manage access to your documentation.
+During the onboarding, you may have noticed the option of making your doc public or private.
```

Comment:
We should probably try to explicit the three types of use cases: public, private and partner.